Functions.Templates/Templates/DurableFunctionsEntity-TypeScript/metadata.json

```
@@ -0,0 +1,18 @@
+﻿{
+  "defaultFunctionName": "DurableFunctionsEntityTS",
```

```suggestion
  "defaultFunctionName": "Counter",
```

Would it make more sense to default to `Counter`?